test/utils_test.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:arweave/src/utils.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  group('Utils:', () {
+    test('longTo8ByteArray', () async {

```

Comment:
Add some tests at the limits

Reply:
[Created tech debt ticket ](https://ardrive.atlassian.net/jira/software/c/projects/PE/boards/2?modal=detail&selectedIssue=PE-595)